public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -316,7 +316,7 @@ export class DashboardPage extends PureComponent<Props, State> {
             scrollTop={updateScrollTop}
             hideHorizontalTrack={true}
             updateAfterMountMs={500}
-            className="custom-scrollbar--page"
+            page={true}
```

Comment:
Outstanding catch, but do you think page a suitable name for this property? I think it seems a bit too generic?